test/e2e/density.go

```diff
@@ -47,11 +47,27 @@ import (
 )
 
 const (
-	MinSaturationThreshold     = 2 * time.Minute
-	MinPodsPerSecondThroughput = 8
-	DensityPollInterval        = 10 * time.Second
+	MinSaturationThreshold        = 2 * time.Minute
+	MinPodsPerSecondThroughput    = 8
+	DensityPollInterval           = 10 * time.Second
+	deploymentResource            = "deployment"
+	replicaSetResource            = "replicaSet"
+	replicationControllerResource = "replicationController"
 )
 
+func runResource(resource string, config testutils.RCConfig) error {
```

Comment:
Or instead of passing an interface{}, create an interface ```SomeGoodNameConfigInterface``` with a single method e.g. ```IsSomeGoodNameConfig``` implemented for our configs and pass this as an argument.